data/seeds/001-users.js

```diff
@@ -0,0 +1,32 @@
+// exports.seed = function(knex, Promise) {
```

Comment:
remember to clean these out when you don't need them any more. Helps to keep the code base clean